com.wireframesketcher.WireframeSketcher.json

```diff
@@ -0,0 +1,81 @@
+{
+  "id": "com.wireframesketcher.WireframeSketcher",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "46",
```

Comment:
47 is the latest. Do you really need the GNOME runtime for a Java app?

Reply:
Yes. It's SWT Java app with native GTK bindings.

Reply:
Since Eclipse provides the low level native bindings, it's hard for me as an application developer that uses their platform to validate their requirements. This is why I am using org.eclipse.Java app as a reference.

I've tried the Freedesktop platform as an experiment and various parts of the application do not work. Some images do not load. Integrated WebKit browser doesn't work.